imports/api/employee.js

```diff
@@ -0,0 +1,209 @@
+    }
+    return db.insert({
+        employeeId, firstName, lastName,
+        // not sure onBoard should be a today or not
+        onBoard: today.getDate(),
+        // should manager id and team id be null at the very begining when meteor doing auth
+        // or it might come with assigned managerId while inserting employee??
+        // I will keep it null for now
+        managerId: null,
+        teamId: null,
+        tasksAssienedId: [],
+        tasksCreatedId: [],
+        tasksWatchingId: [],
+        requestsSentId: [],
+        requestReceivedId: [],
+        commentsId: [],
+        filesId: [],
+        employeesId: [],
+        individualsId: [],
+        teamsId: [],
+        // from my understanding, role should be one of"employer", "employee", "admin"?
+        // i think it would be better to have it once we create employee, since it make sense to determine the level of the account at the time we create it
+        role,
+    }, (err, firstName, lastName) => {
+        if (err) {
+            throw new Meteor.Error(EmployeeError.EMPLOYEE_INSERT_FAIL);
+        } else {
+            console.log(`${
+                EmployeeMessage.EMPLOYEE_CREATED
+            } ${firstName} ${lastName}`);
+        }
+    })
+};
+
+export const removeEmployee = (db, _id) => {
+    if (!isAuthenticated()) {
+        throw new Meteor.Error(AuthError.NOT_AUTH);
+	}
+	//TODO: not too sure use _id or employeeid
+    const employee = Employees.findOne({_id}), {employeeId} = employee;
+    if (! employee) {
+        throw new Meteor.Error(EmployeeError.EMPLOYEE_NOT_EXIST);
+    }
+    if (employeeId !== this.userId) {
+        throw new Meteor.Error(AuthError.NO_PRIVILEGE);
+    }
+
+    // TODO:
+    // need to make sure tasks status associated with this employee has been updated 
+    return db.remove({
+        _id
+    }, err => {
+        if (err) {
+            throw new Meteor.Error(EmployeeError.EMPLOYEE_REMOVE_FAIL);
+        }
+    });
+};
+
+//TODO: do we need this in employee???
+//I think so, delete this for now :/
+// export const createTask = (db, employeeId, taskId, taskTitle, taskDescription) => {
+//     if (!this.userId) {
+//         throw new Meteor.Error(AuthError.NOT_AUTH);
+// 	}
+// 	//TODO: not too sure use _id or employeeid
+// 	const employee = db.findOne({ employeeId });
+// 	if (!employee) {
+// 		throw new Meteor.Error(EmployeeError.EMPLOYEE_NOT_EXIST);
+// 	}
+// 	let { tasksCreatedId } = employee;
+// 	tasksCreatedId = [...tasksCreatedId];
+// 	if (!tasksCreatedId.includes(taskId)) {
+// 		tasksCreatedId = addToList(tasksCreatedId, taskId);
+// 		return db.update({ employeeId }, { tasksCreatedId }, err => {
+// 			if (err) {
+// 				throw new Meteor.Error(EmployeeError.TASK_NOT_CREATABLE);
+// 			} else {
+// 				//TODO:
+// 			}
+// 		});
+// 	}
+    
+//     return db.update(_id, {tasksCreatedId })
+// };
+
+export const removeCreateTaskFromEmployee = (db, employeeId, taskId) => {
```

Comment:
let's call the parameter employeeId just _id instead. In all db, the id for that API are all _id, it's a meteor mongo convention. So when you read the code, you know which id should be used in the db parameter.